test/bigtest/network/scenarios/default.js

```diff
@@ -36,10 +45,279 @@ export default function defaultScenario(server) {
     edition: ''
   });
 
-  server.create('resource', {
-    package: customPackage,
-    title: customTitle
-  });
+  createAccessTypes([
+    {
+      id: '1',
+      type: 'accessTypes',
+      attributes: {
+        name: 'my custom type 1',
+        description: 'description 1',
```

Comment:
no need to manually define all this data. you can use `server.createList('access-type', 10)` which will create 10 access types with random data